test/command_callback/test_llc_command_callback.vader

```diff
@@ -14,6 +14,7 @@ Before:
 After:
   unlet! g:ale_llvm_llc_executable
   unlet! b:ale_llvm_llc_executable
+  delfunction! AssertHasPrefix
```

Comment:
`delfunction!` doesn't work. The command doesn't accept a bang. (`!`)

Reply:
ah, it was my mistake. And `:Vader` did not report the error. I modified commit at 1e72204 and `push -f`ed again.